src/util/suid.c

```diff
@@ -45,6 +45,13 @@ int singularity_suid_init(char **argv) {
 #ifdef SINGULARITY_SUID
     singularity_message(VERBOSE2, "Running SUID program workflow\n");
 
+    singularity_priv_init();
```

Comment:
Hrmm... I don't want to call `singularity_priv_init()` here. Perhaps we should call it right before calling `singularity_suid*()` instead? Thoughts?